java/src/main/kotlin/io/spine/validation/java/setonce/SetOnceJavaConstraints.kt

```diff
@@ -82,7 +83,8 @@ internal sealed class SetOnceJavaConstraints(
     protected val fieldGetterName = "get$fieldNameCamel"
     protected val fieldSetterName = "set$fieldNameCamel"
     protected val fieldGetter = "$fieldGetterName()"
-    protected val declaringMessage = declaredIn.message.javaClassName(declaredIn.fileHeader)
+    protected val declaringMessage =
+        field.declaringType.toMessageType(typeSystem).javaClassName(typeSystem)
```

Comment:
```suggestion
    protected val declaringMessage = field.declaringType
        .toMessageType(typeSystem)
        .javaClassName(typeSystem)
```

For me, it reads a bit easier when formatted this way.